src/isic-client.ts

```diff
@@ -6,9 +6,10 @@ export default class IsicClient extends OauthClient {
   constructor(
     clientId: string,
     isicBaseUrl = 'https://api.isic-archive.com',
+    scopes: string[] = [],
```

Comment:
Actually, should we change this to be the second parameter? Also maybe just make `scopes` required? Or change how optional parameters are passed entirely? Right now, there's no way to accept the default `isicBaseUrl` while explicitly specifying `scopes`.